.editorconfig

```diff
@@ -11,6 +11,7 @@ indent_size = 4
 indent_style = space
 
 # Custom config to override `ktlint` defaults
+ktlint_code_style = intellij_idea
```

Comment:
Ktlint har fått en ny default code style, men tidligere var den dette. Beholder den gamle for å minimere kodeendringer